pkg/handlers/system_intakes.go

```diff
@@ -31,13 +30,7 @@ func (h SystemIntakesHandler) Handle() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		switch r.Method {
 		case "GET":
-			var filterValue models.SystemIntakeStatusFilter
-			statusFilters := r.URL.Query()["status"]
-			if len(statusFilters) == 1 {
-				filterValue = models.SystemIntakeStatusFilter(strings.ToUpper(statusFilters[0]))
-			}
-
-			systemIntakes, err := h.FetchSystemIntakes(r.Context(), filterValue)
+			systemIntakes, err := h.FetchSystemIntakes(r.Context())
```

Comment:
double check - modified API call, so this now fetches all

Reply:
I'm not sure this is in use anymore. I think the [FE fetches intakes through GQL now](https://github.com/CMSgov/easi-app/blob/main/src/components/RequestRepository/index.tsx#L96-L110). @ClayBenson94 should we just toss a comment on here to remove when we remove the REST routes?